selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/SeleniumWebDriverHelper.java

```diff
@@ -45,55 +45,59 @@ protected SeleniumWebDriverHelper(
   }
 
   /**
-   * Set text in provided field with checks
+   * Set text in provided field with checks which get text for compare by {@link
```

Comment:
https://github.com/eclipse/che/pull/8746/commits/fbd23bdd310d9a4558f9bb302d78feaa66da6e1a#r167891133

Reply:
`Ensures {@code element} has got {@code value} in next steps: ensures element is visible and has empty value, sets {@code value} by sending corresponding keys to {@code element} and waits until element has got the {@code value}.`